### PR TITLE
fix: growth of content container to take all the available space

### PR DESCRIPTION
**Pull request by pulkit-30**
fixes: #8369

Issue fix:  To prevent content container growth, to take all the available space.

## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

### Test links

- https://deploy-preview-8409--docusaurus-2.netlify.app/tests/docs/doc-with-another-sidebar

#### After

Screenshot 2022-12-04 at 4 53 17 PM

ref: [changes:](https://deploy-preview-8409--docusaurus-2.netlify.app/tests/docs/doc-with-another-sidebar)

### CODE REVIEW

File: packages/docusaurus-theme-classic/src/theme/Layout/styles.module.css [hunk 1]
```diff
@@ -11,7 +11,7 @@ body {
 }
 
 .mainWrapper {
-  flex: 1 0 auto;
+  flex: 1 0 none;
```
Comment (slorber) [inline packages/docusaurus-theme-classic/src/theme/Layout/styles.module.css:14]: curious, do you have and ref doc mentioning this "none" value?

According to what I find it's only valid when using `flex: 'none'` but not as a 3rd position?

**Review by slorber — CHANGES_REQUESTED** [review]
Thanks for trying

Unfortunately it doesn't work as expected. We want the footer to reach the end of our page here.

CleanShot 2022-12-08 at 16 49 04

**Comment by pulkit-30** [comment]
> Thanks for trying
Unfortunately it doesn't work as expected. We want the footer to reach the end of our page here.

Let me try again!

File: packages/docusaurus-theme-classic/src/theme/Layout/styles.module.css (hunk 1 above)
Comment (pulkit-30) [reply packages/docusaurus-theme-classic/src/theme/Layout/styles.module.css:14]: Sorry, I miss that thing... 
trying again

**Comment by slorber** [comment]
This is a better fix: https://github.com/facebook/docusaurus/pull/8463
